schemas/preferences.sublime-settings.json

```diff
@@ -895,7 +937,8 @@
         },
         "preview_on_click": {
           "markdownDescription": "Preview file contents when clicking on a file in the side bar.\nDouble clicking or editing the preview will open the file and assign it a tab.",
-          "type": "boolean",
+          "type": ["boolean", "string"],
+          "enum": [false, true, "only_left"],
```

Comment:
Interesting use of `enum`. I didn't know you could mix types like that.

Reply:
With a look at `mini_diff` setting we may even be able to omit the `type`.